lib/compute/agent-nodes.ts

```diff
@@ -134,6 +148,37 @@ export class AgentNodes {
         numExecutors: 1,
         amiId: 'ami-07591ca4ef792c2d4',
         initScript: 'echo',
+        remoteFs: 'C:\\Users\\Administrator\\jenkins',
+      };
+
+      this.AL2_X64_EXAMPLE_AGENT = {
+        agentType: 'unix',
+        workerLabelString: 'Jenkins-Example-Agent-X64-C5xlarge-Single-Host',
+        instanceType: 'C54xlarge',
+        remoteUser: 'ec2-user',
+        maxTotalUses: -1,
+        numExecutors: 1,
+        amiId: MachineImage.latestAmazonLinux({
+          generation: AmazonLinuxGeneration.AMAZON_LINUX_2,
+          cpuType: AmazonLinuxCpuType.X86_64,
+        }).getImage(stack).imageId.toString(),
+        initScript: 'sudo yum install -y java-1.8.0-openjdk',
```

Comment:
Do we want to install docker, etc 
Saw this in earlier PR:
```
sudo yum install -y java-1.8.0-openjdk cmake python3 python3-pip && sudo yum groupinstall -y \'Development Tools\' && sudo ln -sfn `which pip3` /usr/bin/pip && pip3 install pipenv && sudo ln -s ~/.local/bin/pipenv /usr/local/bin
```
Your choice.

Reply:
Lets keep it bare minimum and let the user decide what they want to install on the agent.

Reply:
So consider this, main motive of this infrastructure is distribution building. And java, python is required no matter what for this. so above command is minimal but has python so that they can run distribution builds

Reply:
makes sense. Updated the init commands.